utils/roadie-backstage-entity-validator/README.md

```diff
@@ -0,0 +1,128 @@
+# Backstage entity validator
```

Comment:
This doesn't make sense anymore, so maybe rewrite this file?

Reply:
Sure thing